content/e20co/drone-marathon/index.md

```diff
@@ -0,0 +1,103 @@
+---
+author: "dellintosh"
+date: 2017-01-06T18:00:00-06:00
+image: "e20co/drone-marathon"
+repo: "e20co/drone-marathon"
+tags: [ publish, marathon ]
+logo: marathon.svg
+title: marathon
+---
+
+This plugin can be used to deploy applications to a [Marathon](https://mesosphere.github.io/marathon/) server. The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipeline:
+  build:
+    image: nginx:alpine
+  publish:
```

Comment:
maybe consider changing from `publish` to `marathon` in the samples? The name does not really matter, but it might help individuals migrating from 0.4 to 0.5, since `publish` had a special meaning in 0.4